tests/test_s3/test_s3_multipart.py

```diff
@@ -885,3 +885,45 @@ def test_complete_multipart_with_empty_partlist():
     err["Message"].should.equal(
         "The XML you provided was not well-formed or did not validate against our published schema"
     )
+
+
+@mock_s3
+@mock_kms
+def test_ssm_key_headers_in_create_multipart():
+    s3_client = boto3.client("s3", region_name=DEFAULT_REGION_NAME)
+    kms_client = boto3.client("kms", region_name=DEFAULT_REGION_NAME)
+
+    bucket_name = "ssm-headers-bucket"
+    s3_client.create_bucket(Bucket=bucket_name)
+
+    kms_key = kms_client.create_key()["KeyMetadata"]
+    key_name = "test-file.txt"
+
+    create_multipart_response = s3_client.create_multipart_upload(
+        Bucket=bucket_name,
+        Key=key_name,
+        ServerSideEncryption="aws:kms",
+        SSEKMSKeyId=kms_key["KeyId"],
```

Comment:
Can we simplify this to `SSEKMSKeyId="some key"`, without the entire KMS integration? 
It would make the test easier to read, faster, and also make it clear that there is no validation yet

Reply:
done! :+1: